authzed/api/v0/materialize/watchpermissions.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package authzed.api.materialize.v0;
```

Comment:
this package seems right, but the path for this file seems to be `authzed/api/v0/materialize` and not `authzed/api/materialize/v0`